lib/index.js

```diff
@@ -1,8 +1,8 @@
 /*!
- * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2021-2023 Digital Bazaar, Inc. All rights reserved.
  */
-
 import {DidKeyDriver} from './DidKeyDriver.js';
+import {createVerificationSuite} from './utilities.js';
```

Comment:
Does our linter not complain about this? Seems like `c` should come before `D` here ... and I thought it was case insensitive... but maybe not? Could we be missing our linting rules package / it be out of date?

```suggestion
import {createVerificationSuite} from './utilities.js';
import {DidKeyDriver} from './DidKeyDriver.js';
```

Reply:
Yeah, the linter wasn't catching them. I had to update the `eslint-foo` libraries [498b2b2](https://github.com/digitalbazaar/did-method-key/pull/54/commits/498b2b23876c8fdab32217f1b1f479927dd65e78). It catches them now.